app/mailers/application_mailer.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
 class ApplicationMailer < ActionMailer::Base
-  default from: 'from@example.com'
+  default from: 'notifications@jm.bookstore.com'
```

Comment:
I would constantize this. Did you do that by now?
You should add this to the `secrets.yml` file, and then from the secrets, it is fetched from the environment.
Here you have an example: http://guides.rubyonrails.org/upgrading_ruby_on_rails.html#config-secrets-yml

Reply:
Hey Santy, thanks for the idea. Just one thing, Since Rails 5.2 `secrets` are deprecated in favour of `credentials` : https://medium.com/craft-academy/encrypted-credentials-in-ruby-on-rails-9db1f36d8570

Reply:
You are right, Never used the new version of rails yet.
My bad.

Reply:
Anyways if you can, include this from the credentials file